src/dupradar/config.vsh.yaml

```diff
@@ -0,0 +1,104 @@
+name: dupradar
+description: Assessment of duplication rates in RNA-Seq datasets
+keywords: [ rnaseq, duplication, genomics ]
+links:
+  homepage: https://bioconductor.org/packages/release/bioc/html/dupRadar.html
+  documentation: https://bioconductor.org/packages/release/bioc/vignettes/dupRadar/inst/doc/dupRadar.html
+  repository: https://github.com/ssayols/dupRadar
+references: 
+  doi: "10.1186/s12859-016-1276-2"
+license: GPL-3.0
+
+argument_groups:
+- name: Input
+  arguments: 
+  - name: --id
+    type: string
+    description: Sample ID
+  - name: --input
+    type: file 
+    required: true
+    description: Path to input alignment file in BAM format
+  - name: --gtf_annotation
+    type: file 
+    required: true
+    description: Path to GTF annotation file.
+  - name: --paired
+    type: boolean
+    description: Add flag if input alignment file consists of paired reads
+  - name: --strandedness
```

Comment:
What if this parameter is not set?

Reply:
Default value will be 0 (unstranded), I will include this in the modified script